GAE/test/org/akvo/flow/api/app/DataPointServletTest.java

```diff
@@ -219,6 +220,44 @@ public void noAssignmentTest() {
         final Long surveyId = randomId();
         final DataPointServlet servlet = new DataPointServlet();
         final List<SurveyedLocale> foundDataPoints = servlet.getDataPointList(surveyId, randomId());
-        assertNull(foundDataPoints);
+        assertTrue(foundDataPoints.isEmpty());
+    }
+
+    @Test
+    public void testRetrieveDataPointsByLastUpdateTime() throws InterruptedException {
+        final Long surveyId = randomId();
+        final List<SurveyedLocale> oldDataPoints = createDataPoints(surveyId, 5);
+        final Date lastUpdateTime = new Date();
+        Thread.sleep(1500); // only datapoints created after this point should be retrieved
```

Comment:
I don't think we should filter by `lastUpdatedDateTime`, but if we need to. We can create a few datapoints "in the future" directly instead of dealing with `Thread.sleep` ? :thinking: